quipucords/scanner/network/runner/roles/check_dependencies/tasks/main.yml

```diff
@@ -149,21 +149,27 @@
     internal_have_chkconfig: "{{ internal_have_chkconfig_cmd.get('rc') == 0 }}"
   ignore_errors: yes
 
-- name: gather internal_whereis_ifconfig_cmd
-  raw: export LANG=C LC_ALL=C; whereis ifconfig
-  register:  internal_whereis_ifconfig_cmd
+- name: gather internal_have_ifconfig_cmd
+  raw: export LANG=C LC_ALL=C; command -v ifconfig
+  register:  internal_have_ifconfig_cmd
+  become: yes
```

Comment:
Why does this add `become: yes` and `when: 'user_has_sudo'`? Unless I misunderstand something, the way we use `ifconfig` does not actually require `sudo`, and and normal user can probably run `ifconfig -a`.

I noticed you added this sudo requirement in several other places that I don't think require them. Did I overlook an explanation for this?

Reply:
Yes, the main idea is to maintain a consistent environment between place where we call `command -v` and place that is executed conditionally on `internal_have_*` variable. There are few executables that we call as root and as normal user, so they are duplicated.